docs/summary_metrics.md

```diff
@@ -19,16 +20,15 @@ memoryUsage:
     eventId: entity.guid
 ```
  
-A summary metric can be either an **NRDB-query-based metric**, a **tag metric** (summary tag), or a **derived metric**, depending on the source of the information. Therefore, you need to configure exactly ONE out of `query`, `tag` or `derive`. If an amount different to one is detected, the metric definition will not be valid.
+A summary metric can be either an **NRDB-query-based metric**, a **tag metric** (summary tag), or a **golden metric reference**, depending on the source of the information. Therefore, you need to configure exactly ONE out of `query` or `tag`. If an amount different to one is detected, the metric definition will not be valid. `query` can coexist with `goldenMetric` for the time being, but the former is being deprecated in favor of the latter.
```

Comment:
Do we have an estimated ETA for that? Should we only point users to goldenMetric instead?

Reply:
When all summary metrics are migrated to use `goldenMetric`, `query` support will be removed. This should happen soon as it is currently in progress.